### PR TITLE
const BIDI: bool

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

File: src/spi.rs [hunk 1]
```diff
@@ -585,13 +505,13 @@ impl<SPI: Instance, PINS, TRANSFER_MODE, OPERATION> Spi<SPI, PINS, TRANSFER_MODE
 
     /// Return `true` if the TXE flag is set, i.e. new data to transmit
     /// can be written to the SPI.
-    pub fn is_txe(&self) -> bool {
+    pub fn is_tx_empty(&self) -> bool {
```
Comment (therealprof) [inline src/spi.rs:508]: What is the motivation for these renames?

Comment (burrbull) [reply src/spi.rs:508]: This is not principal.
Main motivation is compatibility with f1xx-hal:
https://github.com/stm32-rs/stm32f1xx-hal/blob/6c6c8c5154510b519fdf74207d5ff626accc0e39/src/spi.rs#L450

Comment (burrbull) [reply src/spi.rs:508]: Revert? Or make copy and deprecate?

Comment (therealprof) [reply src/spi.rs:512]: I hadn't thought of that, but copy and deprecate sounds better to me.

Comment (burrbull) [reply src/spi.rs:512]: done. See last commit

**Review by therealprof — APPROVED** [review]
LGTM, thanks.

bors r+

**Comment by burrbull** [comment]
bors retry

**Pull request merged**